client/src/sagas/watchers/selfReview.js

```diff
@@ -32,27 +44,97 @@ function* workerLoadAllUserSelfReviewSaga({ payload }) {
 export function* watchUserSelfReviewSaga() {
   yield takeLatest(LOAD_ALL_USER_SELF_REVIEWS, workerLoadAllUserSelfReviewSaga)
 }
+//Load all self reviews
+function* workerLoadAllSelfReviewSaga() {
+  try {
+    const selfReviews = yield call(loadAllSelfReviews)
+    yield put(setAllSelfReviews(selfReviews.data.data))
+  } catch (e) {
+    if (e.response.data && e.response.data.message) {
+      yield put(setAllSelfReviewsError(e.response.data.message))
+    } else {
+      yield put(setAllSelfReviewsError(e))
+    }
+  }
+}
+
+export function* watchSelfReviewSaga() {
+  yield takeLatest(LOAD_ALL_SELF_REVIEWS, workerLoadAllSelfReviewSaga)
+}
 
 function* workerUpdateUserSelfReviewSaga({ payload }) {
   const { id, body } = payload
   try {
     const selfReviews = yield call(updateSelfReview, id, body)
-    if (selfReviews.data.status == 'error') {
+    if (selfReviews.data.status === 'error') {
       yield put(setUpdateReviewError(selfReviews.data.message))
     }
-    if (selfReviews.data.status == 'success') {
+    if (selfReviews.data.status === 'success') {
       yield put(setUpdateReviewStatus(selfReviews.data.message))
     }
+    const Reviews = yield call(loadAllSelfReviews)
+    yield put(setAllSelfReviews(Reviews.data.data))
   } catch (e) {
     if (e.response.data && e.response.data.message) {
       //To do add code for all api calls .. invalid token case falls here
-      yield put(setAllSelfReviewsForUserError(e.response.data.message))
+      yield put(setUpdateReviewError(e.response.data.message))
     } else {
-      yield put(setAllSelfReviewsForUserError(e))
+      yield put(setUpdateReviewError(e))
     }
   }
 }
 
 export function* watchUpdateUserSelfReviewSaga() {
   yield takeLatest(UPDATE_SELF_REVIEW, workerUpdateUserSelfReviewSaga)
 }
+
+function* workerCreateSelfReviewSaga({ payload }) {
+  const { body } = payload
+  try {
+    const selfReviews = yield call(createSelfReview, body)
+    if (selfReviews.data.status === 'error') {
+      yield put(setSelfReviewError(selfReviews.data.message))
+    }
+    if (selfReviews.data.status === 'success') {
+      yield put(setSelfReviewSuccess(selfReviews.data.message))
+    }
+    const Reviews = yield call(loadAllSelfReviews)
```

Comment:
Please use Camel case for variable name.

Reply:
done